### DISCUSSION TITLE
Hi there [May 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5818] [Discord Threads]
> 🙋‍♂️ **Océane PUECH**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Océane PUECH**  
#### this one works  
![image0](https://cdn.discordapp.com/attachments/1095747438343631059/1095747562360807425/image.png)  
#  
> 🙋‍♂️ **Océane PUECH**  
#### this one not :  
![image0](https://cdn.discordapp.com/attachments/1095747438343631059/1095747713045368923/image.png)  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Hello <@692700658570100787>    
#### Can you share tad more details about your platform? You can copy paste `project info` from project menu (click on project name on top left corner, copy project info)  
#  
> 🙋‍♂️ **Océane PUECH**  
#### Hello <@857998597474484295> ! Sure : Node: **v16.17.1**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### RootDB: **sqlite3**   
#### PackageVersion: **0.104.3**  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Please try on latest NocoDB release. If you still face issue, please help log it in our GitHub repository.    
#### https://github.com/nocodb/nocodb/issues  
#  
> 🙋‍♂️ **Océane PUECH**  
#### I found the problem !    
#### It's because I have "?" in column/property. All my data are "hidden" in this case  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1095747438343631059)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
